tests/test_visitors/test_ast/test_statements/test_wrong_named_keyword.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.refactoring import (
+    WrongNamedKeywordViolation,
+)
+from wemake_python_styleguide.visitors.ast.statements import (
+    WrongNamedKeywordVisitor,
+)
+
+
+@pytest.mark.parametrize('code', [
+    'print(**{"@": "2"})',
+    'print(**{"2ab": "2"})',
```

Comment:
Let's test more: `!`, `&a`, `()`, `-b`, `+1`, `#FFF`

Reply:
And also `"some.value"`, `"Username[FIELD]"`, `"call()"`

Reply:
Done